scripts/update_op_coverage.py

```diff
@@ -124,6 +125,6 @@ def generate_table(build_dir):
   args = parse_arguments()
   content = generate_table(args.build_dir)
   table_path = os.path.join(args.build_dir, 'doc', 'op_coverage.md')
-  with open(table_path, 'w') as f:
+  with open(table_path, 'w', encoding='utf-8') as f:
```

Comment:
fyi @hanhanW, needed this on Windows

Also see the "Note" callout format used above. Trying to use similar styling throughout the published docs. (I'm not totally happy with how they look though, [here](https://google.github.io/iree/GetStarted/LinuxCMake#install-cmake) is a sample).

Reply:
Thanks for fixing this, I missed the encoding issue..